src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/ChallengedAccessDroolsTest.java

```diff
@@ -549,4 +549,123 @@ void shouldRejectAccessFor_ChallengedAccess_ExistingPastEndTime(String roleName,
         assignmentRequest.getRequestedRoles()
             .forEach(roleAssignment -> Assertions.assertEquals(Status.REJECTED, roleAssignment.getStatus()));
     }
+
+    @ParameterizedTest
+    @CsvSource({
+        "SSCS,challenged-access-judiciary,JUDICIAL",
+    })
+    void shouldGrantAccessFor_ChallengedAccess_SSCS_FeePaid(String jurisdiction, String roleName, String roleCategory) {
+        Case caseDetails = caseMap.get(jurisdiction);
+        HashMap<String, JsonNode> roleAssignmentAttributes = new HashMap<>();
+        roleAssignmentAttributes.put("caseId", convertValueJsonNode(caseDetails.getId()));
+        roleAssignmentAttributes.put("requestedRole", convertValueJsonNode(roleName));
+
+        assignmentRequest = TestDataBuilder.buildAssignmentRequestSpecialAccess(
+                "challenged-access",
+                roleName,
+                RoleCategory.valueOf(roleCategory),
+                RoleType.CASE,
+                roleAssignmentAttributes,
+                Classification.PUBLIC,
+                GrantType.CHALLENGED,
+                Status.CREATE_REQUESTED,
+                "anyClient",
+                false,
+                "Access required for reasons",
+                ACTORID,
+                roleAssignmentAttributes.get("caseId").asText() + "/"
+                    + roleAssignmentAttributes.get("requestedRole").asText() + "/" + ACTORID
+            )
+            .build();
+
+
+        FeatureFlag featureFlag = FeatureFlag.builder().flagName(FeatureFlagEnum.SSCS_CHALLENGED_1_0.getValue())
+            .status(true).build();
+
+        featureFlags.add(featureFlag);
+
+        HashMap<String, JsonNode> existingAttributes = new HashMap<>();
+        existingAttributes.put("jurisdiction", convertValueJsonNode(jurisdiction));
+        existingAttributes.put("caseType", convertValueJsonNode(caseDetails.getCaseTypeId()));
+        existingAttributes.put("substantive", convertValueJsonNode("Y"));
```

Comment:
Can you set `substantive` to `"N"` to reflect conditions in PROD. thanks

Reply:
done

Reply:
Resolved in 68ce2df4398504e24354f7bdd011fd09797c323c.